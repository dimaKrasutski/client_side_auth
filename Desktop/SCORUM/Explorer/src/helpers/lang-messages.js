import intlMessagesEN from '../translations/en.yaml';
import intlMessagesZH from '../translations/zh.yaml';
import intlMessagesRU from '../translations/ru.yaml';
import intlMessagesKO from '../translations/ko.yaml';

export const getLangMessages = (lang) => {
  const msg = {
    en: intlMessagesEN,
    zh: intlMessagesZH,
    ru: intlMessagesRU,
    ko: intlMessagesKO,
  };
  return msg[lang];
};
