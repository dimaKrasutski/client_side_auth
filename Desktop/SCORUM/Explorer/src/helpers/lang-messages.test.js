import { getLangMessages } from './lang-messages';

describe('', () => {
  test('getLangMessages', () => {
    expect(getLangMessages('en')).toMatchSnapshot();
  });
});
