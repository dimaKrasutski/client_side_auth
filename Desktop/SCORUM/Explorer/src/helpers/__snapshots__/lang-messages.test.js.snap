// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getLangMessages 1`] = `
Object {
  "explorer.404.button": "Go to homepage",
  "explorer.404.subtitle": "Sorry for inconvenienced",
  "explorer.404.title": "Oops, this page not found",
  "explorer.account-info.balanse-card.title-scr": "SCR Balance",
  "explorer.account-info.balanse-card.title-sp": "SP Balance",
  "explorer.account-info.title": "Account",
  "explorer.account-info.transactions.amount": "Amount",
  "explorer.account-info.transactions.date": "Date",
  "explorer.account-info.transactions.empty.subtitle": "{userName} SCR transaction history will be listed here.",
  "explorer.account-info.transactions.empty.title": "No transactions yet",
  "explorer.account-info.transactions.hash": "Tx hash",
  "explorer.account-info.transactions.title": "Transactions",
  "explorer.account-info.transactions.who": "Who",
  "explorer.block-info.details.date": "Date",
  "explorer.block-info.details.hash": "Hash",
  "explorer.block-info.details.height": "Height",
  "explorer.block-info.title": "Block",
  "explorer.block-info.transactions.all": "All Operations",
  "explorer.block-info.transactions.amount": "Amount",
  "explorer.block-info.transactions.date": "Date",
  "explorer.block-info.transactions.empty.subtitle": null,
  "explorer.block-info.transactions.empty.title": "No transactions in block",
  "explorer.block-info.transactions.from": "From",
  "explorer.block-info.transactions.hash": "Tx hash",
  "explorer.block-info.transactions.title": "Transfers",
  "explorer.block-info.transactions.to": "To",
  "explorer.block-info.transactions.type": "Type",
  "explorer.blocks.table.date": "date",
  "explorer.blocks.table.hash": "hash",
  "explorer.blocks.table.height": "height",
  "explorer.blocks.table.transactions": "â„– of transactions",
  "explorer.blocks.title": "Blocks list",
  "explorer.dashboard.all-blocks": "All blocks",
  "explorer.dashboard.all-transactions": "All transactions",
  "explorer.dashboard.balance-card.total-scr": "Total amount of SCR",
  "explorer.dashboard.balance-card.total-sp": "Total amount of SP",
  "explorer.dashboard.balance-card.total-users": "Total amount of users",
  "explorer.dashboard.blocks.date": "date",
  "explorer.dashboard.blocks.hash": "hash",
  "explorer.dashboard.blocks.height": "height",
  "explorer.dashboard.title": "Dashboard",
  "explorer.dashboard.total-blocks": "Blocks built so far",
  "explorer.dashboard.total-transactions": "Transactions",
  "explorer.dashboard.transactions.date": "date",
  "explorer.dashboard.transactions.hash": "hash",
  "explorer.dashboard.transactions.type": "type",
  "explorer.delegates.table-active.title": "Witnesses",
  "explorer.delegates.table-candidates.title": "Other Candidates",
  "explorer.delegates.table.delegator-name": "Delegator Name",
  "explorer.delegates.table.pos": "pos",
  "explorer.delegates.table.votes": "Votes",
  "explorer.delegates.title": "Active Delegates",
  "explorer.error.unknown": "Oops, something went wrong, please refresh the page.",
  "explorer.menu.blocks": "Blocks",
  "explorer.menu.dashboard": "Dashboard",
  "explorer.menu.delegates": "Delegates",
  "explorer.menu.transactions": "Transactions",
  "explorer.tr-details.amount": "Amount",
  "explorer.tr-details.block": "Block",
  "explorer.tr-details.date": "Date",
  "explorer.tr-details.from": "From",
  "explorer.tr-details.hash": "Hash",
  "explorer.tr-details.memo": "Memo",
  "explorer.tr-details.title": "Transaction details",
  "explorer.tr-details.to": "To",
  "explorer.transactions.empty.subtitle": null,
  "explorer.transactions.empty.title": "No transactions in block",
  "explorer.transactions.title": "Transactions",
  "wallet.error.unknown": "Unknown error",
}
`;
